cl-config.yml

```diff
@@ -0,0 +1,22 @@
+sourceDirectory: 'packages'
+destinationRepo: 'flatfilers/guides'
+destinationDirectory: 'libraries'
+packages:
+  - name: '@flatfile/angular'
```

Comment:
Angular is manually deployed at the moment so is actually `@flatfile/angular-sdk`

Reply:
Ohh, thank you! Fixing!